emission/storage/modifiable/builtin_model_storage.py

```diff
@@ -23,9 +23,13 @@ def upsert_model(self, key:str, model: ecwb.WrapperBase):
         """
         logging.debug("upsert_doc called with key %s" % key)
         entry = ecwe.Entry.create_entry(self.user_id, key, model)
+        ## TODO: Cleanup old/obsolete models
+        # Cleaning up older models, before inserting new model
+        self.trim_model_entries(key)
         logging.debug("Inserting entry %s into model DB" % entry)
         ins_result = edb.get_model_db().insert_one(entry)
-        ## TODO: Cleanup old/obsolete models
+        new_model_count = edb.get_model_db().count_documents({"user_id": self.user_id})
+        logging.debug("New model count for user %s = %s" % (self.user_id, new_model_count))
```

Comment:
I think it would be more useful to log in the change the number of modules - e.g. something like

```suggestion
        logging.debug("model count for user %s has changed %s -> %s" % (self.user_id, old_model_count, new_model_count))
```